src/sedcli_main.c

```diff
@@ -593,7 +593,7 @@ static int handle_setpw(void)
 			sedcli_printf(LOG_ERR, "Unknown error\n");
```

Comment:
looks like this code is repeated multiple times. Maybe create separate function for this?

Reply:
> looks like this code is repeated multiple times. Maybe create separate function for this?

Sounds like a good idea

Reply:
Done